example_site/src/components/ConfigurationControls.css

```diff
@@ -19,3 +19,94 @@ button.configuration-submit-button {
   /* Indent contents of subsection to visually separate them */
   margin-left: 16px;
 }
+
+/* Toggle Switch */
+
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+} 
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+/* Changes background color after click */
+input:checked + .slider {
+  background-color: #ff8c8c;
+}
+
+/* Moves slider to right */
+input:checked + .slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(26px);
+} 
+
+/* Rounded sliders */
+.slider.round {
+  border-radius: 34px;
+}
+
+.slider.round:before {
+  border-radius: 80%;
+} 
+
+/* Alternate "Right" and "Left" text 
+depending on switch toggle */
+.on
+{
```

Comment:
Let's make sure all of the opening brackets are on the same line as the class name!